lib/feature/home/presentation/viewmodel/home_viewmodel.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:file_share/core/services/media_service.dart';
+import 'package:flutter/material.dart';
+import 'package:photo_manager/photo_manager.dart';
+
+class HomeViewmodel {
+  final MediaService _mediaService = MediaService();
+
+  final ValueNotifier<List<AssetEntity>> _photos = ValueNotifier([]);
+  ValueNotifier<List<AssetEntity>> get photos => _photos;
+
+  final ValueNotifier<int> _photosCount = ValueNotifier(0);
+  ValueNotifier<int> get photosCount => _photosCount;
+
+  Future<void> permission() async {
```

Comment:
Methods should have names that read as verbs instead of nouns.
E.g `doSomething` instead of `something`.